crud.py

```diff
@@ -0,0 +1,52 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def get_all_authors(
+        db: Session,
+        skip: int = 0,
+        limit: int = 10
+):
+    return db.query(models.Author).offset(skip).limit(limit).all()
```

Comment:
Ensure that `models.Author` exists in the `models` module and has the necessary attributes like `id`, `name`, and `bio`.